components/calendar/calendar.js

```diff
@@ -633,8 +633,8 @@ class Calendar extends LocalizeCoreElement(RtlMixin(LitElement)) {
 		}
 	}
 
-	getShownValue() {
-		return new Date(this._shownYear, this._shownMonth).toISOString();
+	getShownYearAndMonth() {
+		return { year: this._shownYear, month: this._shownMonth };
```

Comment:
The [mini calendar in the LMS](https://github.com/Brightspace/lms/blob/c1a03638388a6a37386c8cbe1d065072531dfbe5/lp/framework/web/D2L.LP.Web.UI/Desktop/Controls/CalendarMini/CalendarMini.js#L8) is currently using this. Is the plan to switch it over at the same time? Might be easier to add the new method, switch it over, then remove the old to avoid JS errors in the interim.

Reply:
You're right. There was a reason I only added the new method first. Then I completely forgot why I did it. Will revert the last commit.